pancancer_evaluation/utilities/classify_utilities.py

```diff
@@ -66,14 +68,24 @@ def run_cv_cancer_type(data_model, gene, cancer_type, sample_info, num_folds,
                 warnings.simplefilter("ignore")
                 X_train_raw_df, X_test_raw_df = du.split_by_cancer_type(
                    data_model.X_df, sample_info, cancer_type,
-                   num_folds=num_folds, fold_no=fold_no,
-                   use_pancancer=use_pancancer, seed=data_model.seed)
+                   num_folds=num_folds,
+                   fold_no=fold_no,
+                   use_pancancer=use_pancancer,
+                   use_pancancer_only=use_pancancer_only,
+                   seed=data_model.seed)
         except ValueError:
             raise NoTestSamplesError(
                 'No test samples found for cancer type: {}, '
                 'gene: {}\n'.format(cancer_type, gene)
             )
 
+        if X_train_raw_df.shape[0] == 0:
+            # this might happen in pancancer only case
+            raise NoTrainSamplesError(
```

Comment:
If this error is only raised for cancers with 0 samples, maybe update the text to say 0 samples instead of insufficient samples. I at least thought that you had some threshold you were using to determine suffciency